include/nccl_ofi_tracepoint.h

```diff
@@ -27,52 +28,83 @@
 } while(0)
 
 /***** RDMA PROTOCL *****/
+#define N_NVTX_DOMAIN_PER_COMM 8
+
+#define NCCL_OFI_NVTX_TRACE_PER_COMM 1
+#define NCCL_OFI_NVTX_TRACE_PER_DEV 0
```

Comment:
Actually I'm not sure about this, just because of the additional overhead of getting the runtime parameter. Would prefer to keep these macros as lean as possible, since they do seem to affect performance.

Reply:
I'm against making this runtime configurable -- NVTX as a library promises low-overhead in the case that these probes are present and inactive, and implementing our own runtime probe layer atop that is poor design, let's just integrate properly with theirs.

Going back to the original comment, for the bits that are compile-time configurable, they should be integrated with autotools and should be ultimately found in config.h, the same way I originally did `--enable-nvtx`. It's not reasonable to expose this only via manually modifying the header itself and building with a dirty tree.

Reply:
Yeah, you're right. I think I figured out how to do this in Autoconf parlance.

Reply:
Done